specifyweb/frontend/js_src/lib/components/Attachments/Preview.tsx

```diff
@@ -22,7 +22,7 @@ export function AttachmentPreview({
   return (
     <div
       className={`
-        flex items-center justify-center rounded bg-white shadow-xl
```

Comment:
these changes affect both the dialog and the attachment tool. are you sure they are needed?
The bugs Grant reported are related to container shadow, not individual item shadow, right?

Reply:
Dismissing this as the change was intentional